OpenRA.Mods.Common/Effects/GravityBomb.cs

```diff
@@ -72,9 +74,10 @@ public void Tick(World world)
 			velocity -= acceleration;
 			pos += velocity;
 
-			if (pos.Z <= args.PassiveTarget.Z)
+			var dat = world.Map.DistanceAboveTerrain(pos);
+			if (dat.Length <= 0)
```

Comment:
The best option, as @abcdefg30 pointed out, would be to check both `pos.Z <= args.PassiveTarget.Z` *and* `pos.Z <= args.PassiveTarget.Z`